build.gradle.kts

```diff
@@ -65,10 +65,11 @@ spotless {
     }
 
     kotlin {
-        ktlint()
         target("**/*.kt")
+        ktlint()
     }
     kotlinGradle {
+        target("**/*.kts")
```

Comment:
it wasn't linting because of this? gradle :~

Reply:
seems so :(